File: Data Structures/Stack.cpp
#include<bits/stdc++.h>
using namespace std;
int main()
{
	int top=-1,size;
	cout<<"Enter the no. of elements in the stack"<<endl;
	cin>>size;
	int ar[size];
	while(1)
	{
		int choice;
		cout<<endl<<endl<<"---------------------"<<endl;
		cout<<"1. Push element in the stack"<<endl;
		cout<<"2. Pop element in the stack"<<endl;
		cout<<"3. Display all elements in the stack"<<endl;
		cout<<"4. Exit Program"<<endl<<endl;
		cin>>choice;
		switch(choice)
		{
			case 1:
				{
					if(top==size-1)
						cout<<endl<<"Element cannot be Pushed 'Stack is full'"<<endl<<endl;
					else
					{
						cout<<"Enter element to be inserted"<<endl;					
						cin>>ar[++top];
					}
					break;		
				}
			case 2:
				{
					if(top==-1)
						cout<<endl<<"No elements to Pop 'Stack is empty'"<<endl<<endl;
					else
					{
						cout<<endl<<"Poped element is "<<ar[top--]<<endl;
					}
					break;
				}
			case 3:
				{
					if(top==-1)
						cout<<endl<<endl<<"No elements to Display 'Stack is empty'"<<endl<<endl;
					else
					{
						int temp=top;
						cout<<endl<<endl<<"Elements in the stack are "<<endl<<endl;
						while(temp>=0)
						{
							cout<<ar[temp--]<<"  ";
						}
						cout<<endl<<endl;	
					}
					break;
				}
			case 4:
				{
					exit(0);
				}
			default:
			{
				cout<<endl<<endl<<"Incorrect option please try again "<<endl<<endl;
			}	
		}
	}
}

File: Java LAB/NetBeansProjects/Assignment 2/src/Stack.java
import java.util.Scanner;
public class Stack {
    int[] st= new int[5];
    int top;
    Stack()
    {
        top=-1;
    }
    Scanner in = new Scanner(System.in);
    void push()
    {
        if(top==4)
        {
            System.out.println("Stack overflow");
            return;
        }
        int num;
        System.out.println("Enter data");
        num=in.nextInt();
        st[++top]=num;
    }
    void pop()
    {
        if(top==-1)
        {
            System.out.println("Stack underflow");
            return;
        }
        int num;
        num=st[top--];
        System.out.println("Poped element is: "+ num);
    }
    void display(){
        System.out.println("Elements in stack are:");
        for(int i=0;i<=top;i++)
            System.out.print(st[i]+"\t");
    }
    
    public static void main(String [] args)
    {
        int ch=0;
          Scanner in = new Scanner(System.in);
        Stack S1= new Stack();
        while(ch!=4)
        {   
            System.out.println("Enter your choice");
            System.out.println("1. push");
            System.out.println("2. pop");
            System.out.println("3. display");
            System.out.println("4. exit");
            ch=in.nextInt();
            switch(ch)
            {
                case 1: S1.push();
                        break;
                case 2:S1.pop();
                        break;
                case 3: S1.display();
                        break;
                case 4: break;
            }
        }
    }
}
